lib/core/unittest/.gitignore

```diff
@@ -1,2 +1,2 @@
 # Ignore temporary test files
-test.log
+test.log
```

Comment:
There are no changes to this file except removing the newline from the end of the last line.  Please change your editor settings not to do this.  *nix best practice is to have a newline at the end of the last line of the file.  There's an option in the CLion settings for this.